src/frontend/src/utils/gantt.utils.ts

```diff
@@ -92,3 +140,42 @@ export const sortWbs = (a: { wbsNum: WbsNumber }, b: { wbsNum: WbsNumber }) => {
   }
   return aWbsNum.workPackageNumber - bWbsNum.workPackageNumber;
 };
+
+/**
+ * Sort function for the teams on the Gantt chart.
+ * Hardcoding the team names so that we can get the teams in the exact order we want.
+ * However, everything else is not hard coded and allows for more or less teams.
+ * @param a name of team
+ * @param b name of other team
+ * @returns a number for sorting
+ */
+export const sortTeamNames = (a: string, b: string): number => {
```

Comment:
This is kinda weird, could we make this a map of values to values, or assign the specific values to constants or smthn. All these numbers in here just feel a lil like magic numbers.

Reply:
they are definitely magic numbers, it's just forcing the sort function to work a certain way

wym values to values

Reply:
Yeah, I meant teams to values, my b